tests/utils/utils.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect */
+
+'use strict';
+
+describe( 'extend()', function() {
```

Comment:
I'd wrap all of these describes with one more:
```
describe('Utils', ...
```

Then it would produce clearer results, like:
"Utils extend() should extend and override"